ml-proto/README.md

```diff
@@ -168,9 +187,13 @@ cmd:
   ( assert_return_nan (invoke <name> <expr>* ))        ;; assert return with floating point nan result of invocation
   ( assert_trap (invoke <name> <expr>* ) <failure> )   ;; assert invocation traps with given failure string
   ( assert_invalid <module> <failure> )                ;; assert invalid module with given failure string
+  ( input <string> )                                   ;; read script from file
+  ( output <string> )                                  ;; output module to file
 ```
 
-Invocation is only possible after a module has been defined.
+Invocation, most assertions, and output is only possible after a module has been defined.
+
+The input and output commands determine the requested file format from the file name extension. They can handle both `.wast` and `.wasm` files.
```

Comment:
From this description, I might guess that when `input` gets a .wast file, it just executes the whole script, right?